activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Emit structured events for Active Record.
```

Comment:
Thanks for your PR!

I think it doesn't need changelog because it's bug fix and should have been working.

We will need to backport this too. /cc @gmcgibbon

Reply:
I already described in the description of PR, this bug was already fixed in `main` by #55900. 8-1-stable is only branch that we need to fix a bug.